_includes/scenarios/integrate/request-and-response-introduction.md

```diff
@@ -0,0 +1,107 @@
+An example could be a key, coding the content of the Request, which helps to identfify the correct internal process at the time of receiving the Response.
+The metadata content is not processed by enmeshed.
+RequestItems and RequestItemGroups have the required `mustBeAccepted` property.
+If it is enabled, the Request can't be accepted without accepting the respective RequestItems or RequestItemGroups.
+Note that the `mustBeAccepted` property of a RequestItem inside a RequestItemGroup will only be assessed, if the RequestItemGroup itself is accepted.
+By default RequestItems can be accepted automatically.
+However, the sender may enforce a manual acceptance step using `requireManualDecision`.
+Depending on the kind of RequestItem, additional properties may be available.
+
+## Responses
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/b42bc3d1-bb48-4bd5-a645-016dce559b30" id="kCvNDB.IYTUd"></iframe></div>
+
+### Structure of Responses
+
+Once the other Identity receives a Request, it can decide whether or not to accept it.
+If so, a [Response]({% link _docs_integrate/data-model-overview.md %}#response) will be created, containing a [ResponseItem]({% link _docs_integrate/data-model-overview.md %}#responseitem) or [ResponseItemGroup]({% link _docs_integrate/data-model-overview.md %}#responseitemgroup) for every RequestItem or RequestItemGroup, respectively.
+The Response, then, is wrapped in a [ResponseWrapper]({% link _docs_integrate/data-model-overview.md %}#responsewrapper), which holds additional information about the corresponding Request, so that it can be processed correctly once it is returned.
+
+### Types of ResponseItems
+
+There are three different categories of ResponseItems.
+If a RequestItem is accepted, an [AcceptResponseItem]({% link _docs_integrate/data-model-overview.md %}#acceptresponseitem) will be created.
+Depending on the kind of RequestItem, it might be a specific AcceptResponseItem, extending the actual AcceptResponseItem to answer to RequestItems demanding additional information.
+For example, a ReadAttributeRequestItem is accepted using a [ReadAttributeAcceptResponseItem]({% link _docs_integrate/requests-and-requestitems.md %}#readattributerequestitem-response).
+If a RequestItem is rejected, however, a [RejectResponseItem]({% link _docs_integrate/data-model-overview.md %}#rejectresponseitem) is created.
+Lastly, in case the enmeshed Runtime detects a problem, an [ErrorResponseItem]({% link _docs_integrate/data-model-overview.md %}#errorresponseitem) is generated.
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/1d3f3866-4d85-46b5-8523-ecc581052f4b" id="NCvNTKLN71pl"></iframe></div>
+
+### Properties associated with Responses
```

Comment:
Same here, actually Responses are stored within the corresponding LocalRequest's `response` property.